src/components/Registration/ProducerRegistration.js

```diff
@@ -0,0 +1,1202 @@
+          variant="outlined"
+          fullWidth
+          id="fullname"
+          label="Contact Name"
+          name="contactName"
+          value={formState.contactName}
+          disabled
+        >
+          {formState.contactName}
+        </TextField>
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          id="phone"
+          label="Phone"
+          name="phone"
+          value={formState.phone}
+          onChange={handleChange}
+          disabled
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          id="email"
+          label="Email"
+          name="email"
+          value={formState.email}
+          onChange={handleChange}
+          disabled
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextField
+          variant="outlined"
+          required
+          fullWidth
+          id="county"
+          name="county"
+          placeholder="County"
+          value={formState.county}
+          onChange={handleChange}
+        />
+      </Grid>
+      <Grid item xs={2}>
+        <InputLabel id="time-label">Call During</InputLabel>
+      </Grid>
+      <Grid item xs={4}>
+        <TextField
+          id="time"
+          labelId="time-label"
+          name="startTime"
+          onChange={handleChange}
+          type="time"
+          defaultValue="00:00"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          inputProps={{
+            step: 300,
+          }}
+          value={formState.startTime}
+        />
+      </Grid>
+      <Grid item xs={1}>
+        ~
+      </Grid>
+      <Grid item xs={4}>
+        <TextField
+          id="time"
+          name="endTime"
+          onChange={handleChange}
+          type="time"
+          defaultValue="12:00"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          inputProps={{
+            step: 300,
+          }}
+          value={formState.endTime}
+        />
+      </Grid>
+      {errorMsg && <Grid item xs={12} className="error-msg">{errorMsg}</Grid>}
+
+      <Grid item xs={12}>
+        <Button
+          className={classes.greenButtonSmall}
+          type="button"
+          onClick={onNext}
+          color="primary"
+          variant="contained"
+          disabled={step1EmptyInputCheck}
+          center
+        >
+          Next
+        </Button>
+      </Grid>
+    </Grid>,
+  ];
+}
+
+function Step2({
+  currentStep, formState, classes, handleChange, onPrev, errorMsg,
+  onNext, handleBool, step2EmptyInputCheck,
+}) {
+  if (currentStep !== 2) {
+    return null;
+  }
+  return [
+    <div className={classes.subtitleText}>
+      Contact Information
+    </div>,
+    <Grid
+      container
+      spacing={2}
+      alignItems="center"
+    >
+      <Grid item xs={12}>
+        <TextField
```

Comment:
Can we make the website, address line 1, and address line 2 text boxes the same style as the others where the text is in the box and collapses into the top left corner when we click on it?